.jshintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "esversion": 6
```

Comment:
wonder why there's `.jshintrc`? I think you use the eslint

Reply:
Good catch!  I was working with `jshint` and forgot to remove it 👍